babycenter/templatetags/pagination.py

```diff
@@ -0,0 +1,35 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag()
+def get_pages(page, padding=2, edge_padding=2):
+    pages = list(page.paginator.page_range)
+    curr = page.number - 1
+
+    start = []
+    end = []
+    before_current = []
+    if (page.number > edge_padding):
+        start = pages[:edge_padding]
+    if (page.number <= len(pages) - edge_padding):
+        end = pages[-edge_padding:]
+    if page.number == 2:
+        before_current.append(1)
```

Comment:
I don't think this will hold for all values of `padding` and `edge_padding`, for example, if the page number was `3`, `padding` was `3` and `edge_padding` is `3`. I'm busy thinking of a more robust way of us doing figuring out what is in these lists.